### PR TITLE
NodeDefDetails: show taxonomies manager in PanelRight when editing NodeDef

### PR DESCRIPTION
**Pull request by SteRiccio**

### CODE REVIEW

File: webapp/components/Table/Content/Content.js [hunk 1]
```diff
@@ -24,7 +25,7 @@ const Content = (props) => {
   ) : (
     <div className="table__content">
       <div className="table__row-header" style={{ gridTemplateColumns }}>
-        {React.createElement(rowHeaderComponent, props)}
+        {React.createElement(rowHeaderComponent, { props, ...otherProps })}
```
Comment (minotogna) [inline webapp/components/Table/Content/Content.js:28]: so you really need to speed props here?
is so, can they be called rowProps ?

File: webapp/components/survey/NodeDefDetails/TaxonProps.js [hunk 2]
```diff
@@ -49,25 +56,54 @@ const TaxonProps = (props) => {
...
-          </Link>
+          </button>
         </div>
       </FormItem>
+
+      {showTaxonomyPanel && (
+        <PanelRight
+          width="100vw"
+          onClose={async () => {
+            await dispatch(TaxonomyActions.setTaxonomyForEdit(null))
```
Comment (minotogna) [inline webapp/components/survey/NodeDefDetails/TaxonProps.js:90]: do you really need to await here?

File: webapp/loggedin/surveyViews/taxonomy/taxonomyView.scss [hunk 3]
```diff
@@ -1,8 +1,8 @@
-@import "~@webapp/style/vars";
+@import '~@webapp/style/vars';
```
Comment (minotogna) [inline webapp/loggedin/surveyViews/taxonomy/taxonomyView.scss:1]: remove ~@

File: webapp/components/Table/Content/Content.js (hunk 1 above)
Comment (SteRiccio) [reply webapp/components/Table/Content/Content.js:28]: ok

File: webapp/components/survey/NodeDefDetails/TaxonProps.js (hunk 2 above)
Comment (SteRiccio) [reply webapp/components/survey/NodeDefDetails/TaxonProps.js:90]: right, it's not necessary

File: webapp/loggedin/surveyViews/taxonomy/taxonomyView.scss (hunk 3 above)
Comment (SteRiccio) [reply webapp/loggedin/surveyViews/taxonomy/taxonomyView.scss:1]: ok

File: webapp/components/survey/NodeDefDetails/BasicProps/store/useIsDisplayInParentPageDisabled.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import * as NodeDefLayout from '@core/survey/nodeDefLayout'
+
+import { useSurveyCycleKey } from '@webapp/store/survey'
+
```
Comment (minotogna) [inline webapp/components/survey/NodeDefDetails/BasicProps/store/useIsDisplayInParentPageDisabled.js:4]: do really need this hook? u just set a property in the useBasiProps?

File: webapp/components/survey/NodeDefDetails/BasicProps/store/useIsDisplayAsTableDisabled.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
+import * as Survey from '@core/survey/survey'
+import * as NodeDef from '@core/survey/nodeDef'
+
+import { useSurvey } from '@webapp/store/survey'
+
```
Comment (minotogna) [inline webapp/components/survey/NodeDefDetails/BasicProps/store/useIsDisplayAsTableDisabled.js:5]: do really need this hook? u just set a property in the useBasiProps?

File: webapp/components/survey/NodeDefDetails/store/actions/useCancelEdits.js [hunk 6]
```diff
@@ -2,16 +2,16 @@ import { useDispatch } from 'react-redux'
 import { useHistory } from 'react-router'
 
 import { NodeDefsActions } from '@webapp/store/survey'
-import * as NodeDefState from '../state'
+import * as State from '../state'
```
Comment (minotogna) [inline webapp/components/survey/NodeDefDetails/store/actions/useCancelEdits.js:5]: dont you export the State from index?
this should be ```import {State} from '../state'```

File: webapp/components/survey/NodeDefDetails/store/actions/useSaveEdits.js [hunk 7]
```diff
@@ -10,18 +10,18 @@ import { LoaderActions, NotificationActions } from '@webapp/store/ui'
 import * as SurveyState from '@webapp/store/survey/state'
 import * as NodeDefsActions from '@webapp/store/survey/nodeDefs/actions'
 
-import * as NodeDefState from '../state'
+import * as State from '../state'
```
Comment (minotogna) [inline webapp/components/survey/NodeDefDetails/store/actions/useSaveEdits.js:13]: same comment about import

File: webapp/components/survey/NodeDefDetails/store/actions/useSetLayoutProp.js [hunk 8]
```diff
@@ -5,16 +5,16 @@ import * as Survey from '@core/survey/survey'
 import * as NodeDef from '@core/survey/nodeDef'
 
 import { SurveyState } from '@webapp/store/survey'
-import * as NodeDefState from '../state'
-import { useValidateNodeDef } from './useValidateNodeDef'
+import * as State from '../state'
```
Comment (minotogna) [inline webapp/components/survey/NodeDefDetails/store/actions/useSetLayoutProp.js:8]: same comment about State import

File: webapp/components/survey/NodeDefDetails/store/actions/useSetParentUuid.js [hunk 9]
```diff
@@ -1,13 +1,13 @@
 import * as NodeDef from '@core/survey/nodeDef'
 
-import { useValidateNodeDef } from './useValidateNodeDef'
-import * as NodeDefState from '../state'
+import { useValidate } from './useValidate'
+import * as State from '../state'
```
Comment (minotogna) [inline webapp/components/survey/NodeDefDetails/store/actions/useSetParentUuid.js:4]: same comment about State import

File: webapp/components/survey/NodeDefDetails/store/actions/useSetProp.js [hunk 10]
```diff
@@ -10,8 +10,8 @@ import * as NodeDefValidations from '@core/survey/nodeDefValidations'
 import { NotificationActions } from '@webapp/store/ui'
 import { SurveyState } from '@webapp/store/survey'
 
-import { useValidateNodeDef } from './useValidateNodeDef'
-import * as NodeDefState from '../state'
+import { useValidate } from './useValidate'
+import * as State from '../state'
```
Comment (minotogna) [inline webapp/components/survey/NodeDefDetails/store/actions/useSetProp.js:14]: same comment about State import

File: webapp/components/survey/NodeDefDetails/store/actions/useValidate.js [hunk 11]
```diff
@@ -6,9 +6,9 @@ import * as SurveyValidator from '@core/survey/surveyValidator'
 import * as SurveyState from '@webapp/store/survey/state'
 
 import { NodeDefsActions } from '@webapp/store/survey'
-import * as NodeDefState from '../state'
+import * as State from '../state'
```
Comment (minotogna) [inline webapp/components/survey/NodeDefDetails/store/actions/useValidate.js:9]: same comment about State import

File: webapp/components/survey/NodeDefDetails/store/index.js [hunk 12]
```diff
@@ -1,4 +1,5 @@
-import * as NodeDefState from './state'
+import * as State from './state'
```
Comment (minotogna) [inline webapp/components/survey/NodeDefDetails/store/index.js:1]: same comment about State import

File: webapp/components/survey/NodeDefDetails/store/useNodeDefDetails.js [hunk 13]
```diff
@@ -10,8 +10,8 @@ import { useSurvey, useSurveyCycleKey } from '@webapp/store/survey'
 
 import { navigateToChainsView } from '@webapp/loggedin/modules/analysis/chain/actions'
 
-import * as NodeDefState from '../state'
-import { useActions } from '../actions'
+import { useActions } from './actions'
+import * as State from './state'
```
Comment (minotogna) [inline webapp/components/survey/NodeDefDetails/store/useNodeDefDetails.js:14]: same comment about State import

File: webapp/components/survey/NodeDefDetails/BasicProps/store/useIsDisplayInParentPageDisabled.js (hunk 4 above)
Comment (SteRiccio) [reply webapp/components/survey/NodeDefDetails/BasicProps/store/useIsDisplayInParentPageDisabled.js:4]: u r right

File: webapp/components/survey/NodeDefDetails/BasicProps/store/useIsDisplayAsTableDisabled.js (hunk 5 above)
Comment (SteRiccio) [reply webapp/components/survey/NodeDefDetails/BasicProps/store/useIsDisplayAsTableDisabled.js:5]: u r right (2)

File: webapp/components/survey/NodeDefDetails/store/actions/useSetParentUuid.js (hunk 9 above)
Comment (SteRiccio) [reply webapp/components/survey/NodeDefDetails/store/actions/useSetParentUuid.js:4]: ok

File: webapp/components/survey/NodeDefDetails/store/actions/useSetProp.js (hunk 10 above)
Comment (SteRiccio) [reply webapp/components/survey/NodeDefDetails/store/actions/useSetProp.js:14]: ok

File: webapp/components/survey/NodeDefDetails/store/actions/useValidate.js (hunk 11 above)
Comment (SteRiccio) [reply webapp/components/survey/NodeDefDetails/store/actions/useValidate.js:9]: ok

File: webapp/components/survey/NodeDefDetails/store/index.js (hunk 12 above)
Comment (SteRiccio) [reply webapp/components/survey/NodeDefDetails/store/index.js:1]: ok

File: webapp/components/survey/NodeDefDetails/store/useNodeDefDetails.js (hunk 13 above)
Comment (SteRiccio) [reply webapp/components/survey/NodeDefDetails/store/useNodeDefDetails.js:14]: ok

**Pull request merged**